app/controllers/chargeback_assignment_controller.rb

```diff
@@ -4,110 +4,83 @@ class ChargebackAssignmentController < ApplicationController
   after_action :cleanup_action
   after_action :set_session_data
 
-  include Mixins::SavedReportPaging
   include Mixins::GenericSessionMixin
   include Mixins::BreadcrumbsMixin
 
   def self.table_name
     @table_name ||= "chargeback_assignment"
   end
 
-  def tree_select
-    self.x_active_tree = params[:tree] if params[:tree]
-    self.x_node = params[:id]
-    get_node_info(x_node)
-    replace_right_cell
+  def index
+    @title = _("Chargeback Assignments")
+    @tabform = "Compute"
+    session[:changed] = @changed = false
+    build_tabs
+    set_form_vars
+    render :action => "show"
```

Comment:
Umm, now `show` or `index`? Why?

Reply:
@skateman I was trying to follow same pattern as in Configuration controller that is a non-explorer with tabs and forms on each tab. But i will put this back to `index` to not cause any confusion.

Reply:
Yeah, then you don't need an explicit, you just rename the view to `index.html.haml`.